multisig/proposals/vip-002/vip-002-ethereum.ts

```diff
@@ -0,0 +1,579 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setRewardTokenSpeeds(address[],uint256[],uint256[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLastRewardingBlock(address[],uint32[],uint32[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updateJumpRateModel(uint256,uint256,uint256,uint256)", ETHEREUM_MULTISIG],
+    },
+    { target: POOL_REGISTRY, signature: "acceptOwnership()", params: [] },
+    { target: COMPTROLLER_CORE, signature: "acceptOwnership()", params: [] },
+    {
+      target: COMPTROLLER_CORE,
+      signature: "setPriceOracle(address)",
+      params: [RESILIENT_ORACLE],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: ["Core", COMPTROLLER_CORE, "500000000000000000", "1100000000000000000", "100000000000000000000"],
+    },
+    { target: COMPTROLLER_STABLECOINS, signature: "acceptOwnership()", params: [] },
+    {
+      target: COMPTROLLER_STABLECOINS,
+      signature: "setPriceOracle(address)",
+      params: [RESILIENT_ORACLE],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: [
+        "Stablecoins",
+        COMPTROLLER_STABLECOINS,
+        "500000000000000000",
+        "1100000000000000000",
+        "100000000000000000000",
+      ],
+    },
+    { target: COMPTROLLER_CURVE, signature: "acceptOwnership()", params: [] },
+    {
+      target: COMPTROLLER_CURVE,
+      signature: "setPriceOracle(address)",
+      params: [RESILIENT_ORACLE],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addPool(string,address,uint256,uint256,uint256)",
+      params: ["Curve", COMPTROLLER_CURVE, "500000000000000000", "1100000000000000000", "100000000000000000000"],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [WBTC, "30000000", ETHEREUM_MULTISIG],
+    },
+    {
+      target: WBTC,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, 0],
+    },
+    {
+      target: WBTC,
+      signature: "approve(address,uint256)",
+      params: [POOL_REGISTRY, "30000000"],
+    },
+    {
+      target: vWBTC_Core,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["6171"],
+    },
+    {
+      target: POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [vWBTC_Core, "750000000000000000", "800000000000000000", "30000000", TREASURY, "30000000000", "25000000000"],
+      ],
+    },
+    {
+      target: TREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
```

Comment:
We have native ETH in the VTreasury. We should:
1) transfer ETH from the VTreasury to the guardian (function `withdrawTreasuryNative` in the VTreasuryV8 contract)
2) wrap ETH in the WETH contract, to get WETH tokens

There are 10 ETH in the VTreasuryV8: 5 ETH for the ETH market in the core pool, and 5 ETH for the ETH market in the Liquid Staked ETH pool. I would withdraw and wrap the 10 ETH at the same time, to save gas